content/handbook/security/corporate/end-user-services/laptop-management/laptop-wipe/_index.md

```
@@ -0,0 +1,20 @@
+---
+title: ノートパソコンワイプ（ファクトリーリセット）プロセス
+upstream_path: /handbook/security/corporate/end-user-services/laptop-management/laptop-wipe/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-11T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## 概要
+
+ノートパソコンワイプは、IT アナリストとの[アポイントメントの予約](#schedule-a-wipe)を通じて実行する必要があり、マシンをワイプし、ベースのオペレーティングシステムを再インストールし、GitLab が提供したすべてのソフトウェアと設定を削除します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)